### DISCUSSION TITLE
Category field with multi sources

### DISCUSSION THREAD

**Discussion by alexanderbuergin** [craftcms/cms#9767] [Ideas] [answered]
Inside a category field you only can select one source for each field. It would be very useful to have the same option like in the asset or entry field, so you can handle the output via twig. 

See:
<img width="484" alt="Bildschirmfoto 2021-08-27 um 10 38 35" src="https://user-images.githubusercontent.com/85960512/131116197-62c73ed2-0e33-499f-bd30-4f6bfccd8ccf.png">
<img width="580" alt="Bildschirmfoto 2021-08-27 um 10 39 01" src="https://user-images.githubusercontent.com/85960512/131116207-64f9d581-5e35-4432-b9e2-0a18624314a6.png">
<img width="549" alt="Bildschirmfoto 2021-08-27 um 10 28 05" src="https://user-images.githubusercontent.com/85960512/131116209-27641718-a53e-44b7-826e-79ff1168c8ea.png">

**Comment by brandonkelly** [comment, marked as answer]
This was previously requested by #8901.
